components/MessageLoading/index.js

```diff
@@ -0,0 +1,41 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import SVGIcon from "../Icons";
+
+export default function MessageLoading() {
+  return (
+    <MessageContainer>
+      <MessageHeadline>wird geladen...</MessageHeadline>
+      <IconContainer>
+        <SVGIcon variant="refresh" width="80px" />
+      </IconContainer>
+    </MessageContainer>
+  );
+}
+
+const MessageContainer = styled.section`
```

Comment:
Same here for the naming of your styled components as mentioned above.

Reply:
fixed